nifty/pipeline/nifsUtils.py

```diff
@@ -1516,15 +1522,19 @@ def tagFlat(self, cals):
                 if prefix.upper()+"-FLAT" not in hdul['PRIMARY'].header['DATALAB']:
                     hdul['PRIMARY'].header['DATALAB'] = hdul['PRIMARY'].header['DATALAB']+'-'+prefix.upper()+"-FLAT"
                 hdul['PRIMARY'].header['SHFILE'] = (os.path.split(cals.shift_file)[1], self.keywordDict['SHFILE'])
-                hdul['PRIMARY'].header['FCOMBINE'] = (len(cals.flats), self.keywordDict['FCOMBINE'])
-                for i in range(len(cals.flats)):
-                    hdul['PRIMARY'].header['FCMB'+str(i+1)] = (os.path.split(cals.flats[i])[1], self.keywordDict['FCMB'])
+                if len(cals.flats) > 1:
+                    hdul['PRIMARY'].header['FCOMBINE'] = (len(cals.flats), self.keywordDict['FCOMBINE'])
+                    for i in range(len(cals.flats)):
```

Comment:
Confirming that you mean for the primary header to NOT have ACMB?, FCMB?, DCMB? keywords when there's only a single input?